docs/_templates/layout.html

```diff
@@ -4,3 +4,12 @@
     {{ super() }}
 	<a href="{{ pathto('genindex') }}">Keyword Index</a>
   {% endblock %}
+
+  {%- block extrahead %}
+	<script src="https://unpkg.com/wikipedia-preview@1.4.0/dist/wikipedia-preview.production.js"></script>
```

Comment:
Not sure if it's a good idea to have the docs depend on external resources like this. It would be better if it could be downloaded when the docs get built and included in `html_js_files` in `docs/conf.py`.

At the very least this should have the [SRI](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) `integrity` attribute with a hash to ensure that the file that gets served is the one we built the docs with.